README.md

```diff
@@ -51,6 +51,20 @@ Feel free to open issues and/or fork the project to contribute ([see Development
 
 Chrome 48+, Firefox 44+, Safari 8+, IE 10+.
 
+## Semantic Versioning
+
+The dress-code library follows [semantic version standards](https://semver.org/).
+
+However, to be clear, we must describe which part of the exposed API we consider public and which one we consider private.
+When using common programming languages such as Java, PHP or Javascript this concept it's usually defined by the language semantics (eg. private or protected keywords) or by some naming conventions (eg. having an `_underscore` before the name of the function or the variable).
+
+The dress-code library consider part of the **PUBLIC** API just:
+
+* all css class names and selectors exposed by the library
+* all documented [scss](http://sass-lang.com/) variables/placeholder/mixins (we consider a scss variable/mixin/placeholder documented just if it's mentioned in one of the scss references published on the [documentation website](http://zalando.github.io/dress-code/)).
```

Comment:
The API clarification is a very good step. Thank you for that!

Nevertheless the API documentation in the current form has a few issues:
1. it's rather a demo not a reference, it's hard to quickly answer the question "Is variable X a part of API or not?", or "What is API for mixin Y?" Search feature doesn't seem to work for that.
2. it's always latest version so it's not possible to track API changes. After a new release it already points to updated version of API, which in this moment is used by 0% of users (except dresscode developers).

Reply:
@mrac 

Good points!

1. it should be fixed or at least we will work on improving documentation when closing this issue #407 , we are planning to use [sassdoc](http://sassdoc.com/) for that

2. Yeah we might consider to have multiple versions of the documentation, but this it's a little bit trickier with the current setup. In any case since using sassdoc a user can... navigate to the tag and check the comments directly or checkout the tag and run npm start to have an older version of the demo/documentation locally. BTW will be a good idea to open an issue for having browsable versions of the documentation site. Can you do that?